### DISCUSSION TITLE
大佬你好 我想要研究一下CPU和内存调度算法 我用JAVA写的，遇到了一些问题，希望参考一下大佬C语言写的思路

### DISCUSSION THREAD

**Discussion by nyanyaww** [ouuan/ouuan#23] [Q&A (中文)] [answered]
大佬你好 我想要研究一下CPU和内存调度算法 我用JAVA写的，遇到了一些问题，希望参考一下大佬C语言写的思路。就是那个os_bak的项目。https://github.com/wchi-123/fun-project/tree/main/os_bak

**Comment by ouuan** [comment, marked as answer]
你问错地方了，你想问的这个人抄了我的 profile 还没改 link...

https://github.com/wchi-123/wchi-123/tree/cd4dc48e51f6ad4ba5dcb583f4cbca98cce8f944
